climada/engine/impact_calc.py

```diff
@@ -40,7 +40,7 @@ def __init__(self,
                  exposures,
                  impfset,
                  hazard,
-                 imp_mat=None):
+                 imp_mat=sparse.csr_matrix((0, 0))):
```

Comment:
A sparse.csr_matrix is mutable - right?
Hence the default argument should still be None and then reassigned first thing within the method.

Reply:
See 90536e4e96fd67756fcfc0584e4e5a8efcf02d46